homeassistant/components/unifi/sensor.py

```diff
@@ -413,6 +413,20 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda hub, obj_id: f"device_temperature-{obj_id}",
         value_fn=lambda hub, device: device.general_temperature,
     ),
+    UnifiSensorEntityDescription[Devices, Device](
+        key="Device Uplink MAC",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        api_handler_fn=lambda api: api.devices,
+        available_fn=async_device_available_fn,
+        device_info_fn=async_device_device_info_fn,
+        name_fn=lambda device: "Uplink MAC",
+        object_fn=lambda api, obj_id: api.devices[obj_id],
+        unique_id_fn=lambda hub, obj_id: f"device_uplink_mac-{obj_id}",
+        supported_fn=lambda hub, obj_id: "uplink_mac"
```

Comment:
Lambdas should not be multi line, if it doesn't fit on one line make it a function instead.